Libraries/Components/TextInput/TextInput.js

```diff
@@ -492,6 +492,12 @@ const TextInput = React.createClass({
     inlineImagePadding: PropTypes.number,
 
     /**
+     * Allow TextInput to pass touch event to parent.
+     * @platform ios
```

Comment:
It does not make any difference on Android, where the default behaviour is for a TextInput to pass the touch event to the parent.